backend/app/services/repositories/user_file_repository.py

```
@@ -0,0 +1,86 @@
+        try:
+            user_file = UserFile(
+                user_id=user_id,
+                file_type=file_type,
+                file_name=file_name,
+                file_key=file_key,
+                file_size=file_size,
+            )
+            self.db.add(user_file)
+            self.db.commit()
+            self.db.refresh(user_file)
+            return user_file
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)